packages/extension/src/shared/token/type.ts

```diff
@@ -18,6 +18,7 @@ export interface Token extends Required<RequestToken> {
   verified?: boolean
 }
 
-export interface TokenListToken extends Token {
-  updatedAt?: number
+export interface TokenListTokens {
+  updatedAt?: number,
```

Comment:
It seems this field always exists, not optional?

Reply:
Not for the initial alphTokens, otherwise yes

Reply:
Okay, then let's keep it optional